gridstatus/ercot.py

```diff
@@ -1773,6 +1777,59 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+
+        # just so we can share parse logic
+        kwargs = dict(
+            report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+            date=date,
+            verbose=verbose,
+            extension="csv",
+        )
+
+        if date == "latest":
+            doc = self._get_document(**kwargs)
+            docs = [doc]
+        else:
+            docs = self._get_documents(**kwargs)
+
+        df = self._handle_sced_system_lambda(docs)
+        return df
+
+    def _handle_sced_system_lambda(self, docs):
```

Comment:
this should take a `verbose` parameter. 

before reading the URL, add logging statement.

also wrap the loop in tqdm for a progress bar. helping since a day downloads many files 

here is an example: https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L950